README.md

```diff
@@ -1,3 +1,15 @@
+# Spring Cloud Azure Samples
+- This is a sample project for demonstration purposes.   
+- The project demonstrates how to use [Spring Cloud Azure](https://microsoft.github.io/spring-cloud-azure/current/reference/html/index.html) to develop applications.    
+- The sample project contains 42 samples, and it still grows.    
+- These samples are grouped by Azure services and [Spring Cloud Azure libraries](https://github.com/Azure/azure-sdk-for-java/tree/feature/azure-spring-cloud-4.0/sdk/spring).    
+
+    > For example: [single-namespaces](servicebus/spring-cloud-azure-starter-integration-servicebus/single-namespaces), it located in the path `/servicebus/spring-cloud-azure-starter-integration-servicebus`.
```

Comment:
The "[single-namespaces]" is abrupt here, readers can get lost. And by the way, it should be single-namespace, without s.